src/MigrationsConfigurationFactory.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Roave\PsrContainerDoctrine;
+
+use Doctrine\Migrations\Configuration\Configuration;
+use Psr\Container\ContainerInterface;
+
+/**
+ * @method Configuration __invoke(ContainerInterface $container)
+ */
+class MigrationsConfigurationFactory extends AbstractFactory
+{
+    /**
+     * @inheritDoc
+     */
+    protected function createWithConfig(ContainerInterface $container, string $configKey)
+    {
+        $migrationsConfig = $this->retrieveConfig($container, $configKey, 'migrations_configuration');
```

Comment:
I would vote for a shorter convention here:  `migrations` rather than `migrations_configuration` for more consistency with other top level configration keys (`driver`, `connection`, `cache` etc..)

Reply:
Aye, sounds good. Also, do we need top-level documentation for these keys? Maybe under `example/`?

Reply:
Yes, It would be more complete with entries in `example/full-config.php`. Not sure if an additional dedicated example such as `example/migrations-config.php` would be nicer.

This addition also worth to mention in `Readme.md` imo.